softgroup/basis/common-protocol/src/main/java/com/softgroup/common/protocol/ActionHeaderFactory.java

```diff
@@ -0,0 +1,17 @@
+package com.softgroup.common.protocol;
+
+/**
+ * @author AlexKazmirchuk
+ * @since 30.03.17.
+ */
+public class ActionHeaderFactory {
+    public static ActionHeader createHeader(ActionHeader requestHeader){
```

Comment:
@AlexKazmirchuk It is bad practice do make copy methods. Use copy constructor instead